metrics/prometheus.go

```diff
@@ -0,0 +1,111 @@
+package metrics
+
+import (
+	"strings"
+	"sync/atomic"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const namespace = "moira"
+
+func NewPrometheusRegistry(registry *prometheus.Registry, service string) Registry {
+	return &PrometheusRegistry{registry, service}
+}
+
+type PrometheusRegistry struct {
+	registry *prometheus.Registry
+	service  string
+}
+
+func (source *PrometheusRegistry) NewTimer(path ...string) Timer {
+	//TODO Is a histogram with default buckets fine?
```

Comment:
I think, yes, let's leave the default bucket for now.
I hope someday we will document the Moira metrics, there we will write all these features and bring them to one form.

Reply:
Deal